packages/zowe-explorer/src/job/ZosJobsProvider.ts

```diff
@@ -387,11 +387,12 @@ export class ZosJobsProvider extends ZoweTreeProvider implements IZoweTree<IZowe
         let favJob: ZoweJobNode;
         if (contextValue.startsWith(globals.JOBS_JOB_CONTEXT)) {
             favJob = new ZoweJobNode({
-                label,
+                label: "",
                 collapsibleState: vscode.TreeItemCollapsibleState.Collapsed,
                 parentNode,
                 job: new JobDetail(label),
             });
+            favJob.label = label;
```

Comment:
While this is a creative workaround to avoid the bug, I feel that we should address the underlying issue in the `ZoweJobNode` class (or the more generic `ZoweTreeNode` if applicable) - it looks like the label is mishandled in one of these constructors.